src/pybind/mgr/dashboard_v2/__init__.py

```diff
@@ -27,3 +27,6 @@ def __getattr__(self, item):
     logging.root.handlers[0].setLevel(logging.DEBUG)
     os.environ['PATH'] = '{}:{}'.format(os.path.abspath('../../../../build/bin'),
                                         os.environ['PATH'])
+    import sys
+    import mock
+    sys.modules['ceph_module'] = mock.Mock()
```

Comment:
Can you add a comment why you need to mark this module as loaded? Does it make sense to relocate this to the RBD mirroring unit test?

Reply:
See also [This PR](https://github.com/openattic/ceph/pull/85/files#diff-b82e11902c1a4d3d8ad2c8d811c328c4R31). This module is needed for all tests that import modules that import `ceph_module`. If your code actually uses `ceph_module`, you won't get any data from it, thus you have to mock all calls anyway. This line mainly fixes import statements, which are executed really early. I'd leave it here.